### PR TITLE
refactor(Picker): Refactoring Picker component with FloatingUI

### PR DESCRIPTION
**Pull request by VadymBezpalko**
Resolves: #777 #474 #903 #349

## Description
Refactor topics:

- Using FloatingUI to set element position, handle clicks, keyboard evenets (dissmissing, list navigation)
- Adding Virtuoso virtualization to the Picker list
- Improving keyboard navigation
- Improving SelectAll mechanics and styling

## Storybook

https://feature-777--613a8e945a5665003a05113b.chromatic.com/?path=/story/components-picker--default

## Checklist

**Obligatory:**

- [x] Self review (use this as your final check for proposed changes before requesting the review)
- [x] Add reviewers (`livechat/design-system`)
- [x] Add correct label
- [x] Assign pull request with the correct issue

### CODE REVIEW

File: packages/react-components/src/components/Picker/Picker.tsx [hunk 1]
```diff
@@ -1,98 +1,39 @@
 import * as React from 'react';
 
-import cx from 'clsx';
-
-import { Size } from 'utils';
-
-import { KeyCodes } from '../../utils/keyCodes';
-
+import { cx } from '@emotion/css';
```
Comment (marcinsawicki) [inline packages/react-components/src/components/Picker/Picker.tsx:3]: Why `@emotion/css` instead of `clsx`?

**Review by marcinsawicki — APPROVED** [review]

**Review by vladko-uxds — APPROVED** [review]
🔥

File: packages/react-components/src/components/Picker/Picker.tsx (hunk 1 above)
Comment (VadymBezpalko) [reply packages/react-components/src/components/Picker/Picker.tsx:3]: Auto import by IDE - will fix this 👍

File: packages/react-components/src/components/Picker/components/PickerListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    return (
+      <>
+        {item.showCheckbox && (
+          <input
+            type="checkbox"
+            className={styles[`${itemClassName}__checkbox`]}
+            checked={isSelected}
+          />
```
[the remark this answers is not in the document]
Comment (VadymBezpalko) [reply packages/react-components/src/components/Picker/components/PickerListItem.tsx:53]: I mainly dealt with the component rendering logic - I didn't even see this input 😄

**Pull request merged**